qa/L1_tensorflow-dali_test/test.sh

```diff
@@ -11,12 +11,15 @@ NUM_GPUS=$(nvidia-smi -L | wc -l)
 
 CUDA_VERSION=$(nvcc --version | grep -E ".*release ([0-9]+)\.([0-9]+).*" | sed 's/.*release \([0-9]\+\)\.\([0-9]\+\).*/\1\2/')
 # from 1.13.1 CUDA 10 is supported but not CUDA 9
-# MPI is present in CUDA 10 image already so no need to build it
 if [ "${CUDA_VERSION}" == "100" ]; then
     pip install tensorflow-gpu==1.13.1
 else
     pip install tensorflow-gpu==1.12
+fi
 
+# MPI might be present in CUDA 10 image already so no need to build it if that is the case
+if [ ! -d "/usr/local/mpi" ]; then
+    apt-get update && apt-get install -y wget
```

Comment:
What if MPI is present, but not in `/usr/local//mpi`? Is it possible?
Or `/usr/local/mpi` turns out to be empty? ;)

Reply:
Then the test will misbehave. 
Do you have a suggestion for a more appropriate check?

Reply:
Maybe just `which ompi_info`?

Reply:
```
if ! [ -x "$(command -v mpicxx)" ]; then
```
works